src/components/Tabs.jsx

```
@@ -1,39 +1,99 @@
+            </button>
+          ))}
+        </div>
+        <div className="tab-content">
+          {state.error ? (
+            <p className="error">{state.error}</p>
+          ) : state.data[activeTab] ? (
+            <>
+              <h2 className="title">Title {activeTab + 1}</h2>
+              <div
+                className="text"
+                dangerouslySetInnerHTML={{ __html: state.data[activeTab] }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)